bindings/pyroot/pythonizations/python/ROOT/_pythonization/_ttree.py

```diff
@@ -145,15 +145,61 @@ def _TTree__iter__(self):
     if bytes_read == -1:
         raise RuntimeError("TTree I/O error")
 
-def _SetBranchAddress(self, *args):
-    # Modify the behaviour if args is (const char*, void*)
-    res = SetBranchAddressPyz(self, *args)
+def _SetBranchAddress(self, bname, addr, *args, **kwargs):
```

Comment:
Why `**kwargs`?

Reply:
It's a minor convenience fix for the user. Like this, the optional additional arguments can also be passed as keyword arguments, if for the user it makes the code more readable.
https://root.cern.ch/doc/master/classTTree.html#a63d63d79134a65f3ce452994df5bf425